src/config/index.ts

```diff
@@ -168,6 +170,16 @@ export const REVIEW_OWNER_INDEX_PARAM = "?index=";
 export const UPDATE_REVIEW_OWNERS_PARAMS = REVIEW_OWNER_INDEX_PARAM + ID;
 export const ROUTE_PARAM_TRANSACTION_ID = "transactionId";
 export const ROUTE_PARAM_SUBMISSION_ID = "submissionId";
+export const LANDING_PAGE_QUERY_PARAM = "start";
```

Comment:
Moved these constants here as they are URL related and nothing to do with the data model (at least not at present). Has lead to some more minor changes in this PR, due to changed imports etc. Discussed with @markpit and we thought it was better to do this sooner rather than later.